### PR TITLE
Sign out dialog

### PR DESCRIPTION
**Pull request by OwaisAD**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ dist/
 /node_modules/*
 /server/src/generated/*
 
+
```
Comment (Abed01-lab) [inline .eslintignore:8]:
```suggestion

```

File: client/src/components/Navigation.tsx [hunk 2]
```diff
@@ -98,15 +108,28 @@ export const Navigation = () => {
...
-            <NavButton
-              text="Sign out"
-              icon="signOut"
-              iconSize="medium"
-              id="signOutBtn"
-              onClick={handleSignOut}
-              isNavBarExpanded={isNavBarExpanded}
-            />
+
+            {/* LOWER NAV BUTTONS */}
```
Comment (Abed01-lab) [inline client/src/components/Navigation.tsx:112]:
```suggestion
```

File: client/tailwind.config.js [hunk 3]
```diff
@@ -1,13 +1,36 @@
-const defaultTheme = require("tailwindcss/defaultTheme");
 /** @type {import('tailwindcss').Config} */
-export default {
-  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
-  darkMode: "class",
+const defaultTheme = require("tailwindcss/defaultTheme");
+module.exports = {
+  darkMode: ["class"],
+  content: ["./pages/**/*.{ts,tsx}", "./components/**/*.{ts,tsx}", "./app/**/*.{ts,tsx}", "./src/**/*.{ts,tsx}"],
```
Comment (Abed01-lab) [inline client/tailwind.config.js:5]: Either specify the folders or include the whole src not both of them

File: client/tsconfig.json [hunk 4]
```diff
@@ -18,7 +18,10 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+
+    /*SHADCN*/
```
Comment (Abed01-lab) [inline client/tsconfig.json:23]:
```suggestion
```

**Review by Abed01-lab — COMMENTED** [review]
Minor changes

File: client/src/components/Navigation.tsx (hunk 2 above)
Comment (OwaisAD) [reply client/src/components/Navigation.tsx:112]: fixed

File: client/tailwind.config.js (hunk 3 above)
Comment (OwaisAD) [reply client/tailwind.config.js:5]: fixed

**Review by Abed01-lab — DISMISSED** [review]
LGTM 🎸

**Pull request merged**